.github/workflows/upload-assets-to-cdn.yml

```diff
@@ -0,0 +1,45 @@
+# Upload files in assets/product-icons & assets/legal to stage and prod S3 buckets for CDN distribution
+#
+# Requirements:
+# 1. Github OIDC Provider in destination AWS account - https://github.com/aws-actions/configure-aws-credentials
+# 2. IAM Role with write access to destination bucket paths, scoped to the repository running the action
+
+name: "CDN Asset Upload"
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'assets/**/'
+      - '!assets/other/**'
+
+jobs:
+  Upload-assets-to-CDN:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Configure Stage AWS credentials
+        uses: aws-actions/configure-aws-credentials@master
+        with:
+          aws-region: us-west-2
```

Comment:
Hmm, I thought stage would be in us-east-1... or does this refer to sts?

Reply:
In this case the S3 bucket & IAM role but both are global, it complained without a region but changing it to us-east-1 for consistency might be good.

Reply:
S3 is "global-ish" -- if you want to receive S3 events, the receiver has to be in the same region the bucket was configured in. (I got bit on a lambda with this.) So using the origin region might preserve the hair of an sre/dev down the road 😁

Reply:
Sounds good to me! 

Updated stage region to us-east-1.